packages/medusa/src/services/order-edit.ts

```diff
@@ -312,4 +313,75 @@ export default class OrderEditService extends TransactionBaseService {
       await orderEditRepo.remove(edit)
     })
   }
+
+  async decline(
+    orderEditId: string,
+    declinedReason: string | undefined,
+    customerId?: string
```

Comment:
suggestion: I think that @srindom mention that it could be a feature that an admin user can also decline. also, for consistency could we declare the third parameter like that
```ts
context: { loggedInUserId: string }
```
In which later on we would be able to pass the transaction as well as part of the context